### PR TITLE
chore: Update docs for v0.6

### PR DESCRIPTION
**Pull request by spotandjake**
This pr updates the website for the changes made in `v0.6`

### CODE REVIEW

File: src/getting_grain.md [hunk 1]
```diff
@@ -22,30 +22,30 @@ _The `--no-quarantine` flag will avoid having to approve the binary in the Secur
 
 ### MacOS x64 - Download
 
-If you'd prefer not to use homebrew, you can [download it](https://github.com/grain-lang/grain/releases/download/grain-v0.5.13/grain-mac-x64) directly from GitHub or using `curl`.
+If you'd prefer not to use homebrew, you can [download it](https://github.com/grain-lang/grain/releases/download/grain-v0.6.0/grain-mac-x64) directly from GitHub or using `curl`.
```
Comment (phated) [inline src/getting_grain.md:25]: This is done automatically by the release

File: src/guide/data_types.md [hunk 2]
```diff
@@ -1,22 +1,82 @@
...
+
+subtractOne(counter)
+
+print(counter) // { count: 1 }
+```
 
 ## Enum Types
 
-Enums allow you to represent data that has multiple variations. Combined with pattern matching, they will make you feel like you've got superpowers. They're the backbone of programming in Grain, and you'll come to find that they make it ridiculously easy to understand what a program does.
+Enums allow you to represent data that has multiple variations. Combined with pattern matching, they will make you feel like you've got superpowers. They're the backbone of programming in Grain, and you'll come to find that they are a very useful tool for expressing complex scenarios intuitively. Among other benefits, they enable Grain to have safe nullability throug the `Option` type and intuitive errors-as-values through the `Result` type (both of which we will cover more extensively in later sections).
```
Comment (phated) [inline src/guide/data_types.md:65]:
```suggestion
Enums allow you to represent data that has multiple variations. Combined with pattern matching, they will make you feel like you've got superpowers. They're the backbone of programming in Grain, and you'll come to find that they are a very useful tool for expressing complex scenarios intuitively. Among other benefits, they enable Grain to have safe nullability through the `Option` type and intuitive errors-as-values through the `Result` type (both of which we will cover more extensively in later sections).
```

File: src/guide/functions.md [hunk 3]
```diff
@@ -62,3 +100,124 @@ let rec fibonacci = (n) => {
...
+```grain
+module Main
+
+let isEven = n => {
+  if (n % 2 == 0) {
+    return true
+  }
+  return false
+}
+
```
Comment (phated) [inline src/guide/functions.md:117]:
```suggestion
```

**Review by phated — APPROVED** [review]
I think we just need to revert the "getting grain" changes and this should be good to go. One thing to note, we need to follow up the release with another docs update that updates the stdlib sidebar to include all the modules we added! We can prepare it early but shouldn't merge until the release updates the docs.

**Comment by ospencer** [comment]
Reverted the changes to getting grain

**Pull request merged**